src/System.Management.Automation/commands/utility/FormatAndOutput/out-console/OutConsole.cs

```diff
@@ -102,6 +98,7 @@ protected override void ProcessRecord()
         {
             if (Transcript)
             {
+                Host.UI.TranscribeOnly = true;
```

Comment:
Looking at the code, the only thing `TranscribeOnly` does is [not write to the console] (https://github.com/PowerShell/PowerShell/blob/02b5f357a20e6dee9f8e60e3adb9025be3c94490/src/System.Management.Automation/commands/utility/FormatAndOutput/out-console/ConsoleLineOutput.cs#L324) so I mispoke when I said `pipeline` (should have said `host`).